packages/probing-response-updater/test/service.test.ts

```diff
@@ -0,0 +1,103 @@
+import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
+import { responseStatus } from "pagopa-interop-probing-models";
+import { createApiClient } from "../../probing-eservice-operations/src/model/generated/api.js";
+import {
+  EserviceService,
+  eServiceServiceBuilder,
+} from "../src/services/eserviceService.js";
+import {
+  AppError,
+  apiUpdateResponseReceivedError,
+  makeApplicationError,
+} from "../src/model/domain/errors.js";
+import { config } from "./../src/utilities/config.js";
+import { mockApiClientError } from "./utils.js";
+
+const apiClient = createApiClient(config.operationsBaseUrl);
+
+describe("eService service test", () => {
+  let eserviceService: EserviceService;
+
+  beforeAll(() => {
+    eserviceService = eServiceServiceBuilder(apiClient);
+  });
```

Comment:
this is probably enough
```suggestion
  const eserviceService: EserviceService = eServiceServiceBuilder(apiClient);
```